.github/scripts/trymerge.py

```diff
@@ -996,8 +1008,25 @@ def find_matching_merge_rule(pr: GitHubPR,
     """Returns merge rule matching to this pr or raises an exception"""
     changed_files = pr.get_changed_files()
     approved_by = set(pr.get_approved_by())
-    rules = read_merge_rules(repo, pr.org, pr.project)
-    reject_reason = f"PR {pr.pr_num} does not match merge rules"
+    issue_link = get_new_issue_link(
+        org=pr.org,
+        project=pr.project,
+        labels=["module: ci"],
+    )
+    reject_reason = (f"No rules find to match PR, "
+                     f"please [report]{issue_link} this issue to DevX team")
+
+    try:
+        rules = read_merge_rules(repo, pr.org, pr.project)
+    except Exception as error:
+        # Any error here means that the script couldn't load the merge rules for
+        # some reasons, and the reject message should say so
+        raise RuntimeError(f"{reject_reason}: {error}")
```

Comment:
Please don't  having a generic exception catching is a bad practice (i.e. this function should already raise correct errors
```suggestion
    rules = read_merge_rules(repo, pr.org, pr.project)
```